src/index.js

```diff
@@ -12,9 +12,7 @@ export { default as VmsPageHeader } from './components/VmsPageHeader'
 
 export * from './helpers'
 
-// TODO: consider prefixes or wrapping in a class, split & directory structure
-export * from './vm-actions'
```

Comment:
Well we will have probably relatively much more action creators than reducers and I would very much like to avoid listing them individually here. For now we can maintain backwards compatibility by turning this line into `export * from './actions'` and later perhaps we can think of how to restructure/namespace the imports better overall.